src/preact.d.ts

```diff
@@ -33,18 +33,23 @@ declare namespace preact {
 		componentDidUpdate?(previousProps:PropsType,previousState:StateType,previousContext:any):void;
 	}
 
+	interface FunctionalComponent<PropsType, StateType> {
```

Comment:
I don't know enough about TS to say for sure, but I think this should context? The signature for Pure Functional Components is `fn(props, context) => jsx`